packages/terra-functional-testing/src/commands/utils/downloadScreenshots.js

```diff
@@ -0,0 +1,51 @@
+const extract = require('extract-zip');
+const fs = require('fs-extra');
+const path = require('path');
+const https = require('https');
+const { Logger } = require('@cerner/terra-cli');
+
+const logger = new Logger({ prefix: '[terra-functional-testing:downloadScreenshots]' });
+
+/**
+ * Downloads and unzips the screenshots from provided url.
+ * @param {string} screenshotUrl - The url to the repository that stores the screenshots.
+ */
+async function downloadScreenshots(screenshotUrl) {
+  if (!screenshotUrl) {
+    logger.error('A url to the screenshot registry is required to download screenshots.');
+    return;
+  }
+
+  await new Promise((resolve, reject) => {
+    https.get(screenshotUrl)
+      .on('response', async (response) => {
```

Comment:
I'm not very familiar with the https api - do we need to handle a situation where an error occurs before the `response` event is triggered?

Reply:
It should always return a response with a statusCode which will indicate whether or not it succeeded or not.